version/news/NEWS-2024.12.1-kousa-dogwood.md

```diff
@@ -41,4 +43,4 @@
 ### Dependencies
 
 ### Deprecated / Removed
-- Added a warning about the deprecation of publishing to Posit Cloud. More information: https://pos.it/pc-publishing (rstudio-pro##7008)
\ No newline at end of file
+- Added a warning about the deprecation of publishing to Posit Cloud. More information: https://pos.it/pc-publishing (rstudio-pro##7008)
```

Comment:
I'm confused as to what changed on this line 🤔 
But should there be a double-hash in the issue number?

```suggestion
- Added a warning about the deprecation of publishing to Posit Cloud. More information: https://pos.it/pc-publishing (rstudio-pro#7008)
```